package.json

```diff
@@ -92,5 +92,11 @@
   },
   "engines": {
     "node": ">=14"
+  },
+  "overrides": {
+    "react": "17.0.1"
+  },
+  "resolutions": {
```

Comment:
What tool is leveraging this? Are we using `npm-force-resolutions`?

Reply:
I think this can be removed

Reply:
This is for yarn, but not entirely sure whether we _really_ need this. We have this in `pluggable-widgets-tools` in case users use `yarn`, but i think the monorepo itself is geared towards `npm`.

Reply:
I agree, we explicitly support npm. I have doubts about supporting another dependency-management tool, as this opens up a whole bunch of maintenance.

Reply:
I have removed it from the root package.json. Should I remove it from the piw-tools?

Reply:
Just leave that for now since that's already inside piw-tools for some time